http/server.go

```diff
@@ -48,7 +47,8 @@ func New(ctx context.Context, client drand.PublicClient, version string, logger
 	//TODO: aggregated bulk round responses.
 	mux.HandleFunc("/public/latest", handler.LatestRand)
 	mux.HandleFunc("/public/", handler.PublicRand)
-	mux.HandleFunc("/group", handler.Group)
+	// NOTE: group is being kept because of retro compatibility
```

Comment:
How so? It has been deleted above.

Reply:
it's because I forgot to remove the comment :)